microsoft-edge/progressive-web-apps-chromium/how-to/widgets.md

```diff
@@ -0,0 +1,495 @@
+          "sizes": "600x400",
+          "label": "The PWAmp mini-player widget"
+        }
+      ],
+      "icons": [
+        {
+          "src": "./favicon-16.png",
+          "sizes": "16x16"
+        }
+      ],
+      "auth": false,
+      "update": 100
+    }
+  ]
+}
+```
+
+In the above example, a music player application defines a mini player widget. The required and optional fields of a widget definition in the web app manifest are documented below:
+
+| Field | Description | Required |
+|:--- |:--- |:--- |
+| `name` | The title of the widget, presented to users. | Yes |
+| `short_name` | An alternative short version of the name. | No |
+| `description` | A description of what the widget does. | No |
+| `icons` | An array of icons to be used for the widget. | Yes |
+| `screenshots` | An array of screenshots that show what the widget looks like. Analogous to the [`screenshot` manifest member](https://developer.mozilla.org/docs/Web/Manifest/screenshots). | Yes |
+| `tag` | A string used to reference the widget in the PWA service worker. | Yes |
+| `template` | The template to use to display the widget in the operating system widgets dashboard. Note: although `template` is a required field, its value is currently not used. See `ms_ac_template` below. | Yes |
+| `ms_ac_template` | The URL to the custom Adaptive Cards template to use to display the widget in the operating system widgets dashboard. See [Define a widget template](#define-a-widget-template) below. | Yes |
+| `data` | The URL where the data to fill the template with can be found. For now, this URL is required to return valid JSON. | Yes |
+| `type` | The MIME type for the widget data. Note that it is, for now, required to be `application/json`. | Yes |
+| `auth` | A boolean indicating if the widget requires authentication. | No |
+| `update` | The frequency, in seconds, at which the widget will be updated. Note that the widget is not updated automatically and you need code in your service worker to do this. See [Access widget instances at runtime](#access-widget-instances-at-runtime). | No |
+
+
+<!-- ====================================================================== -->
+## Define a widget template
+
+To make widgets easy to create and adapt to various operating system widgets dashboards, they are displayed using templates. Two types of templates exist:
+
+* Generic templates, defined by their names using the `template` field.
+* Custom templates, defined by their URLs using a custom template field.
+
+For the time being, only custom Adaptive Cards templates are supported. Adaptive Cards is an open card exchange format that can be used to exchange UI content in a common and consistent way. See [Adaptive Cards Overview](/adaptive-cards/).
+
+To define a custom Adaptive Cards template on Windows 11, use the `ms_ac_template` field in the widget definition that's in your web app manifest. Note that even if `template` is not used at the moment, it is a required field.
```

Comment:
```suggestion
To define a custom Adaptive Cards template on Windows 11, use the `ms_ac_template` field in the widget definition that's in your web app manifest. Although `template` is not currently used, it's a required field.
```
"if" is ambiguous/misleading.  'template' is definitely not used.
https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/i/if-vs-whether-vs-when